terraform/helm/fullnode/values.yaml

```diff
@@ -17,7 +17,7 @@ rust_log: info
 
 image:
   repo: aptoslabs/validator
-  tag: devnet
+  tag: devnet_1a56a55c6c149ccc8d6a38903e42e9626fb10b06
```

Comment:
thanks for the change! could you please keep the tag at devnet to be more generic? devnet tag does get refreshed with all the latest devnet branch updates